mssql-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,13 @@
+driverName=sqlserver
+databaseName=temp1
+selectQuery=select * from ${schema}.${table}
+sourceRef=source
+targetRef=target
+sourceTable=SOURCETABLE
```

Comment:
We should ideally not put the table names here, to ensure that this does not conflict with any other run.